app.js

```diff
@@ -125,37 +125,22 @@ app.get('/data/bootstrapcdn.json', function (req, res) {
 
 // Merge in variable options for sitemap.
 function sitemapOptions (options) {
-    options.http = 'https';
-
     if (env === 'production') {
```

Comment:
The check returns `options` unchanged for production... I could rewrite the whole thing as...

```javascript
// Merge in variable options for sitemap.
function sitemapOptions (options) {
    if (env !== 'production') {
        options.route = {
            '/': {
                disallow: true
            }
        };
    }
    return options;
}
```

Which is cleaner for sure.